app/robots/iCubGenova09/oculusConfig.ini

```diff
@@ -45,9 +46,10 @@ rightCameraPort         /icub/cam/right
 [include TORSO_RETARGETING "torsoRetargeting.ini"]
 
 [OCULUS]
```

Comment:
@S-Dafarra  Here instead of modifying the values of the `OCULUS` group, what we can do is to add `OPENXR` group and keep oculus group as it was and based on  `useOpenXr` we read one of the two options. In this case, if we want to do demo with iCub3 and oculus for any reason, we do not need to change these variables again.

Reply:
That's agood point!

Reply:
I have added this with the last commit. There are some other changes to the last file because of the clang formatting.
The two changes are in https://github.com/robotology/walking-teleoperation/pull/69/files#diff-18df315094c6bba8b4871f9c64c45a56e4ffb8e8c1e79fafedb34731d9dcef89 and https://github.com/robotology/walking-teleoperation/pull/69/files#diff-e53bdd7b4385b92cf93f7c48da41177f28a31e43275cc157c2f05c51c585e649R337-R339